nodeApps/generateSignedFile.js

```diff
@@ -68,18 +37,24 @@ const getSignature = (file, callback) => {
   })
 }
 
+const getCommitMessagesSinceLastRelease = (versionedFile, signature, callback) => {
+  utils.getCommitMessagesSinceLastRelease((err, notes) => {
+    callback(err, notes, versionedFile, signature)
+  })
+}
+
 const getSignedStagingFile = getSignature.bind(null, `${distPath}/${name}.staging.min.js`)
 const getSignedProductionFile = getSignature.bind(null, `${distPath}/${name}.min.js`)
 
-const updateChangelog = (versionedFile, signature, callback) => {
+const updateChangelog = (notes, versionedFile, signature, callback) => {
   fs.readFile('CHANGELOG.md', 'utf-8', (readErr, contents) => {
     if (readErr) return callback(readErr)
     const file = ('' + versionedFile).replace(distPath, '')
     const existingLines = new RegExp(`.*${file}.*`, 'g')
     const newContents = contents
       .replace(existingLines, '')
       .replace(/\n\s*\n/g, '\n')
-      .replace('# Changelog', `# Changelog \n\n- ${file} - signature: ${signature}`)
+      .replace('# Changelog', `# Changelog \n\n- ${file} - signature: ${signature}${notes}`)
```

Comment:
Haven't tested this but shouldn't there be a new line here `\n` or do we want the message to be on the same line?

Reply:
This method is called twice, once with notes, once without, we only want the newline when we have the release notes, so we add the newline to the release notes...
```
notes = `\n  - something here\n  - another comment\n`
```